### DISCUSSION TITLE
Pure WebGPU renderer for JSCAD?

### DISCUSSION THREAD

**Discussion by ziriax** [jscad/OpenJSCAD.org#1254] [General]
First of all, what a great project, thanks!

I'm thinking about integrating this in my hobby eductional reactive programming project [ViKiD]( https://vikid.net)

I would use WebGPU for rendering. Do you know if a pure WebGPU renderer for JSCAD already exists? 

I prefer not to use high level frameworks like ThreeJS or BabylonJS, because I don't have a single stage, stages are generated on the fly.

**Comment by hrgdavor** [comment]
There is an effort to integrate jscad with ThreeJs and Babylon on top of regl, so in that sense it is possible.
